src/containers/pages/JurisdictionAssignment/JurisdictionAssignmentView/tests/index.test.tsx

```diff
@@ -0,0 +1,293 @@
+        url: `${ASSIGN_JURISDICTIONS_URL}/${plan.identifier}`,
+      },
+    };
+
+    const wrapper = mount(
+      <Provider store={store}>
+        <Router history={history}>
+          <ConnectedJurisdictionAssignmentView {...props} />
+        </Router>
+      </Provider>
+    );
+
+    await act(async () => {
+      await new Promise(resolve => setImmediate(resolve));
+      wrapper.update();
+    });
+
+    expect(fetch.mock.calls.length).toEqual(4);
+
+    // check renderer error message
+    expect(wrapper.text()).toMatchSnapshot('should be plan error page');
+  });
+  it('single jurisdiction error Message', async () => {
+    const plan = plans[0];
+    fetch
+      .once(JSON.stringify(plan), { status: 200 })
+      .once(JSON.stringify([]), { status: 500 })
+      .once(JSON.stringify(mtendere), { status: 200 })
+      .once(JSON.stringify(lusaka), { status: 200 });
+    const props = {
+      history,
+      location: {
+        hash: '',
+        pathname: `${ASSIGN_JURISDICTIONS_URL}/${plan.identifier}`,
+        search: '',
+        state: {},
+      },
+      match: {
+        isExact: true,
+        params: { planId: plan.identifier },
+        path: `${ASSIGN_JURISDICTIONS_URL}/:planId`,
+        url: `${ASSIGN_JURISDICTIONS_URL}/${plan.identifier}`,
+      },
+    };
+
+    const wrapper = mount(
+      <Provider store={store}>
+        <Router history={history}>
+          <ConnectedJurisdictionAssignmentView {...props} />
+        </Router>
+      </Provider>
+    );
+
+    await act(async () => {
+      await new Promise(resolve => setImmediate(resolve));
+      wrapper.update();
+    });
+
+    // check renderer error message
+    expect(wrapper.text()).toMatchSnapshot('should be jurisdiction error page');
+  });
+
+  it('getting root jurisdiction error Message', async () => {
+    const plan = plans[0];
+    fetch
+      .once(JSON.stringify(plan), { status: 200 })
+      .once(JSON.stringify([akros2]), { status: 200 })
+      .once(JSON.stringify(mtendere), { status: 200 })
+      .once(JSON.stringify({}), { status: 500 });
+    const props = {
+      history,
+      location: {
+        hash: '',
+        pathname: `${ASSIGN_JURISDICTIONS_URL}/${plan.identifier}`,
+        search: '',
+        state: {},
+      },
+      match: {
+        isExact: true,
+        params: { planId: plan.identifier },
+        path: `${ASSIGN_JURISDICTIONS_URL}/:planId`,
+        url: `${ASSIGN_JURISDICTIONS_URL}/${plan.identifier}`,
+      },
+    };
+
+    const wrapper = mount(
+      <Provider store={store}>
+        <Router history={history}>
+          <ConnectedJurisdictionAssignmentView {...props} />
+        </Router>
+      </Provider>
+    );
+
+    await act(async () => {
+      await new Promise(resolve => setImmediate(resolve));
+      wrapper.update();
+    });
+
+    // check renderer error message
+    expect(wrapper.text()).toMatchSnapshot('should be also be jurisdiction error page');
```

Comment:
Seems we may need to fix this sentence `should be also be jurisdiction error page`

Reply:
yeah